ios-showcase-template.xcodeproj/project.pbxproj

```diff
@@ -524,7 +524,7 @@
 				TargetAttributes = {
 					B3E21B3E1FAC956E00082CF7 = {
 						CreatedOnToolsVersion = 9.1;
-						ProvisioningStyle = Manual;
+						ProvisioningStyle = Automatic;
```

Comment:
That's the major change here. Rest of the values do not matter and are automatically generated by xcode.